### PR TITLE
modified top lvl README to include setup

### PR DESCRIPTION
**Pull request by jordaniza**
Added an additional section to the top level README file to make onboarding slightly easier. Any comments most welcome

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,3 +87,49 @@ Refer to the [Bounty Board Commands and Workflow](https://bankless.notion.site/T
...
+### Configure local `.env` file
+
+The react application looks for an environment variables file named `.env.local` on starting. You can copy the `.env.qa` file for most of the variables. You will need to add the `DEV_MONGODB_URI` and `DEV_DEV_DISCORD_BOUNTY_BOARD_WEBHOOK` as covered below.
+### Setup MongoDB
+
+Connection to MongoDB is handled through the Mongoose DRM. You can either connect to a hosted instance of MongoDB, or run a local development copy.
+
+If running locally, your `.env.local` will contain something like:
+```
+DEV_MONGODB_URI=mongodb://localhost:27017/bountyboard
```
Comment (amaredeus) [inline README.md:111]: We've made a migration from netlify to vercel which allows us to store environments grouped by code level (production, deployment, preview). As such we've renamed the environment variable DEV_MONGODB_URI ->

MONGODB_URI=mongodb://localhost:27017/bountyboard

File: README.md [hunk 2]
```diff
@@ -86,3 +87,49 @@ Refer to the [Bounty Board Commands and Workflow](https://bankless.notion.site/T
...
+```
+If connecting to a remote mongo server, your connection string will be in the format:
+```
+DEV_MONGODB_URI=mongodb://username:password@host:port/bountyboard
```
Comment (amaredeus) [inline README.md:115]: MONGODB_URI

File: README.md [hunk 3]
```diff
@@ -86,3 +87,49 @@ Refer to the [Bounty Board Commands and Workflow](https://bankless.notion.site/T
...
+```bash
+$ mongoimport\
+    --db bountyboard\
+    --collection bounties\
+    --file path/to/mongo/bounties/file.json\
+    --jsonArray # only needed if loading an array
+```
+If you've made it this far, the application should run and should be showing a bounty on the main screen. You can directly query the API backend through the app at `localhost:3000/api/bounties`
+### Setting Up the Discord Webhook
+The `DEV_DEV_DISCORD_BOUNTY_BOARD_WEBHOOK` is not required to start the app, but can be fetched from a member of the bounty board development team. Add it to your `.env.local` file once you have it.
```
Comment (amaredeus) [inline README.md:135]: DISCORD_BOUNTY_BOARD_WEBHOOK

**Review by amaredeus — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jordaniza) [reply README.md]: Cool, have made changes as requested into the next commit in the same PR

**Comment by jordaniza** [comment]
@amaredeus all requested amendments to the prefixes have been made in the latest commits.

**Review by amaredeus — APPROVED** [review]
Very nice thank you!

**Pull request merged**